lib/matplotlib/rcsetup.py

```diff
@@ -1017,16 +1021,16 @@ def validate_hist_bins(s):
     'axes3d.grid': [True, validate_bool],  # display 3d grid
 
     #legend properties
-    'legend.fancybox': [False, validate_bool],
+    'legend.fancybox': [True, validate_bool],
 
     # at some point, legend.loc should be changed to 'best'
-    'legend.loc': ['upper right', validate_legend_loc],
+    'legend.loc': ['best', validate_legend_loc],
 
     # this option is internally ignored - it never served any useful purpose
     'legend.isaxes': [True, validate_bool],
 
     # the number of points in the legend line
-    'legend.numpoints': [2, validate_int],
+    'legend.numpoints': [1, validate_int],
     # the number of points in the legend line for scatter
     'legend.scatterpoints': [3, validate_int],
```

Comment:
yeah, I thought this is what was meant by changing the number of points in the legend.